package/src/languageFeatures.ts

```diff
@@ -886,7 +886,7 @@ export class CompletionAdapter implements monaco.languages.CompletionItemProvide
                     const fullURL = linkWithoutPostfix.startsWith('https')
                         ? linkWithoutPostfix
                         : `${documentationBaseUrl}/${linkWithoutPostfix}`;
-                    return monaco.Uri.parse(fullURL);
+                    return monaco.Uri.parse(`${fullURL}${documentationSuffix || ''}`);
```

Comment:
IMO this function is a little hard to read right now. I'd write it more like this:
```ts
get(_target, prop, _receiver) {
    // The link comes with a postfix of ".md" that we want to remove
    let url = prop.toString().replace('.md', '');
    // Sometimes we get the link as a full URL. For example in the main doc link of the item
    if (url.startsWith('https')) {
        url = `${documentationBaseUrl}/${url}`;
    }
    if (documentationSuffix) {
        url += documentationSuffix;
    }
    return monaco.Uri.parse(url);
}
```